trigger/src/main/scala/core/models/enums/JobStatuses.scala

```diff
@@ -0,0 +1,22 @@
+package core.models.enums
+
+object JobStatuses {
+
+  sealed abstract class JobStatus(val name: String, val isFinalStatus: Boolean) {
+    override def toString: String = name
+  }
+
+  case object InQueue extends JobStatus("InQueue", false)
```

Comment:
Maybe extracting the literals into a companion object or something similar? Could make it easier to reference in other parts of the code (e.g. tests, API, etc).

Reply:
String literal wont be used.  Only case object will be used. There is no point to extract it.